.gitlab-ci.yml

```
@@ -89,9 +89,19 @@ check-web-wasm:
   script:
     # WASM support is in progress. As more and more crates support WASM, we
     # should add entries here.
```

Unrelated, but can we add this link here (or around?)

```suggestion
    # should add entries here. See https://github.com/paritytech/substrate/issues/2416
```

if somebody stumbles on this they would go to the issue. There they would be able to get the context and/or express their feelings.